### ISSUE TITLE
Add support for CRS

### ISSUE THREAD

**Issue by liberostelios** [tudelft3d/azul#6]
It would be useful to add support for CRS when reading a CityJSON (or CityGML) file.

This is really important in cases where horizontal and vertical coordinates are using different units, for instance EPSG:4979 where X and Y are in degrees and Z is in meters (also related to issue #4).

**Comment by liberostelios** [comment]
That's true. I already did that, but it's kind of a mess to try and find an appropriate projection for the dataset.

**Comment by agilliland** [comment]
selfishly I would also like to see CRS support added in, but I fully understand that the app is much simpler without it and it could be a pain to add in and maintain.

I'm curious if instead of fully trying to support CRS identification and transformation if it's perhaps possible to support a basic transform of just the Z value of each coordinate?  I would imagine this being a setting applied by choice after a file is opened similar to the way `show edges` or `show bounding box` are applied.

For example, in the case of EPSG:4979 where X and Y are in degrees and Z is in meters, to transform the Z into degrees so that Azul will render everything properly you simply do `Z/111139.0` which converts meters to degrees.  Maybe there is an easy way to expose something like this so that users put in a coefficient and the Z coordinates are transformed?

Just thinking out loud.

**Comment by kenohori** [comment]
Hello! It's been a long time, but I've been recently spending some time to improve azul and I wanted to get back to this idea.

In short, I didn't want to integrate something like PROJ with all of its dependencies. So, I implemented a bit of a hack to transform degrees to meters in https://github.com/tudelft3d/azul/commit/3b2499d5d640e32a185f8cf0a5243eab948dca99. This is just activated for a few hardcoded CRS.

<img width="912" height="796" alt="Image" src="https://github.com/user-attachments/assets/a7961929-ccf3-43c9-abd4-6a536c358ff3" />
